packages/database-integrations/README.md

```
@@ -0,0 +1,221 @@
+  - `type DatabaseIntegrationTypeWithSslSupport` – Database integration types that support SSL (subset of `DatabaseIntegrationType`)
+  - `type SqlIntegrationType` – SQL integration types (subset of `DatabaseIntegrationType`), excludes `mongodb`
+- runtime values
+  - `databaseIntegrationTypes` – Array of all supported database integration types (`DatabaseIntegrationType[]`)
+  - `databaseIntegrationTypesWithSslSupport` – Array of all database integration types that support SSL (`DatabaseIntegrationTypeWithSslSupport[]`)
+  - `sqlIntegrationTypes` – Array of all SQL integration types (`SqlIntegrationType[]`)
+
+### Integration metadata
+
+- zod schemas
+  - `databaseMetadataSchemasByType` – Zod schemas for validating integration metadata by integration type
+    ```ts
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)